pyproject.toml

```diff
@@ -11,7 +11,7 @@ repository = "https://github.com/NHSDigital/proxygen-cli"
 proxygen = 'proxygen_cli.cli.command_main:main'
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = "^3.9"
```

Comment:
This code actually breaks if you run it with Python 3.8 as it uses the dictionary union operator.